code/datums/id_trim/centcom.dm

```diff
@@ -94,6 +103,9 @@
 /// Trim for Centcom Commanders. All Centcom and Station Access.
 /datum/id_trim/centcom/commander
 	assignment = JOB_CENTCOM_COMMANDER
+	honorific = "Commander"
```

Comment:
The random name generator for ERT's make them get "Commander" and such as their actual names, so you probably should remove those from their name pool if you want them to have honorifics as well.

Reply:
i too fear Commander Commander

Reply:
Sounds like the working name for Cobra Commander.

Reply:
> The random name generator for ERT's make them get "Commander" and such as their actual names, so you probably should remove those from their name pool if you want them to have honorifics as well.

The CentCom honorifics have less selectable positions they can use for this exact reason, but I will confess I haven't yet tested this. If it does actually mess with the names, I'll find a solution.